os/se/ameba/security_ameba_wrapper_tz.c

```diff
@@ -355,8 +384,9 @@ int se_ameba_hal_rsa_sign_md(hal_rsa_mode mode, hal_data *hash, uint32_t key_idx
 	int ret;
 	inout_struc in_output;
 
-	if (key_idx >= KEY_STORAGE_INDEX_MAX) {	/* Index Out of Range */
-		return HAL_INVALID_SLOT_RANGE;
+	ret = se_ameba_index_chk(key_idx);
```

Comment:
same key_idx check missing?

Reply:
All key index check will be done in se_ameba_index_chk().